components/Article/Page.js

```diff
@@ -123,8 +141,32 @@ export const withCommentData = graphql(
 )
 
 const dynamicComponentRequire = createRequire().alias({
-  'react-apollo': reactApollo,
-  'graphql-tag': graphqlTag
+  'react-apollo': {
+    // Reexport react-apollo
+    // (work around until all dynamic components are updated)
+    // ApolloContext is no longer available but is exported in old versions of react-apollo
+    ApolloConsumer,
+    ApolloProvider,
+    Query,
+    Mutation,
+    Subscription,
+    graphql,
+    withQuery,
+    withMutation,
+    withSubscription,
+    withApollo,
+    getDataFromTree,
+    renderToStringWithData,
+    compose
+  },
+  // Reexport graphql-tag to be used by dynamic-components
+  'graphql-tag': {
+    gql,
+    resetCaches,
+    disableFragmentWarnings,
+    enableExperimentalFragmentVariables,
+    disableExperimentalFragmentVariables
+  }
```

Comment:
The default / main export needs to be `gql`. The other exports were never used in dynamic components. IMO we could just do:
```suggestion
  'graphql-tag': gql
```

With the current solution I get an error on `/2020/10/28/sag-wie-hast-du-s-mit-der-frist`:
<img width="948" alt="Screenshot 2021-09-24 at 11 13 39" src="https://user-images.githubusercontent.com/410211/134650304-4d8f9593-11cf-44b2-b946-703d609a7af6.png">